spaces/S000165/properties/P000210.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000165
+property: P000210
+value: true
+refs:
+  - mathse: 5105500
+    name: What properties does the one-point compactification of the Arens-Fort space have?
+---
+
+See {{mathse:5105500}}.
```

Comment:
```suggestion
$X$ is covered by some cofinite sets, each satisfying {P210} with the subspace topology.

Namely, using the notation from {{mathse:5105500}}:
- $X\setminus\{\infty\}$: {S23|P210},
- $X\setminus\{0\}$: {S20|P210}.
```
This seems easier, using what is already known to pi-base.
It's my last comment :-)